apps/condo/next.config.js

```diff
@@ -21,6 +22,7 @@ const withTM = withTMModule([
     '@emotion/styled',
 ])
 
+// TODO(antonal): move config parsing and implement validation in `parseEnvConfigs.js`
```

Comment:
You should provide task number if you want to leave a todo

Reply:
Done